assets/css/style.css

```diff
@@ -0,0 +1,86 @@
+h1 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #227452;
+}
+
+p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  margin: 2em 24px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url("../../assets/img/background-image-mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.logo-placeholder {
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.toolbar {
+  padding: 0 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+}
+
+.menu {
+  padding: 1em;
+  background: none;
+  border: 10px;
+}
+
+.headline button {
+  display: flex;
+  flex-direction: row;
+  background: none;
+  align-items: flex-start;
+  padding: 8px;
+  border: 1px;
+  border-radius: 32px;
+}
+
+.flex-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.contact {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  list-style: none;
```

Comment:
- [ ] Please 🙏🏽 center the social icons to look similar to the Figma template. **Hint:** use padding.

Reply:
Hello @franclobo, thanks for the help, I think I just need to fix the background, could you guide me a bit please? Thank you!